templates/leptos-ssr/content/src/server.rs

```diff
@@ -1,19 +1,21 @@
 use leptos::ServerFn;
-use spin_sdk::http::{ResponseOutparam, IncomingRequest};
+use leptos_spin::{render_best_match_to_stream, RouteTable, server_fn::register_explicit};
+use spin_sdk::http::{IncomingRequest, ResponseOutparam};
 use spin_sdk::http_component;
-use leptos_spin::{render_best_match_to_stream, RouteTable};
 
 #[http_component]
-async fn handle_{{project-name | snake_case}}(req: IncomingRequest, resp_out: ResponseOutparam) {
+async fn handle_{{crate_name}}(req: IncomingRequest, resp_out: ResponseOutparam) {
```

Comment:
I think you need to retain the `snake_case` filter here.  Also does `crate_name` work?  `project-name` is provided from the `spin new` command.